tests/console/consoletest_finish.pm

```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Cleanup and switch (back) to X11
+# Maintainer: Alberto Planas <aplanas@suse.com>
```

Comment:
Please update to someone recently working on this module. I don't mind if you add me, i.e. "okurz@suse.de"